controllers/cinder_controller.go

```diff
@@ -138,7 +137,7 @@ func (r *CinderReconciler) Reconcile(ctx context.Context, req ctrl.Request) (res
 		r.Client,
 		r.Kclient,
 		r.Scheme,
-		r.Log,
+		log,
```

Comment:
I really like the idea of adding the logger function to the helper !!!

But can we call it Log (capital 'L')? See how it will be referenced in a comment below.